game/src/ai.ts

```diff
@@ -0,0 +1,99 @@
+import { Game } from "./game";
+import { GameState, Move } from "./types";
+
+function calculateBestMove(
```

Comment:
I'm going to merge this  & make a few changes but since you requested a review I'm going to leave a few comments here!